apps/monitoring/version-reporter/helm-charts.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: version-reporter-service-helmcharts
+  namespace: monitoring
+  labels:
+    app: version-reporter-service
+spec:
+  schedule: "*/10 * * * *" # interval would be reviewed after testing complete
+  concurrencyPolicy: "Forbid"
+  failedJobsHistoryLimit: 5
+  startingDeadlineSeconds: 300
+  successfulJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      backoffLimit: 3
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: version-reporter-service-sa
+          containers:
+            - name: version-reporter-service-helmcharts
+              image: sdshmctspublic.azurecr.io/hmcts/version-reporter-services:master-helmcharts #{"$imagepolicy": "flux-system:version-reporter"}
+              imagePullPolicy: Always
+              env:
+                - name: ENVIRONMENT
+                  value: "ptlsbox"
+                - name: CLUSTER_NAME
```

Comment:
May need to use some other means to do this as this could change, i may drop this variable as its just extra info and do  `cluster=$(kubectl config current-context)` within the application do get same info

Reply:
What is the need/erquirement for this? Is this on a per different environment?

Do we maybe need one base file with multiple patch files?

Reply:
Ok, that will be a better approach or if easier to get the current cluster detail then one less property to manage.